walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,25 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+    
+    String greeting = greeter.greet("   Ada   ");
+    
+    // Whitespace should be trimmed
```

Comment:
Nit: since you picked a good test name ("testGreetingTrimsWhitespace"), this comment isn't really necessary